### PR TITLE
use valid values in values.yaml

### PR DESCRIPTION
**Pull request by MajdT51**

### CODE REVIEW

File: charts/managed-identity-wallets/templates/_helpers.tpl [hunk 1]
```diff
@@ -70,6 +70,13 @@ Usage: include "postgresContext" (list $ "your_include_function_here")
 {{- include $function (dict "Values" $.Values.postgresql "Chart" (dict "Name" "postgresql") "Release" $.Release) }}
 {{- end }}
 
+{{/*
+Create the default JDBC url
+*/}}
+{{- define "managed-identity-wallets.jdbcUrl" -}}
+{{- printf "jdbc:postgresql://%s-postgresql:5432/postgres?user=%s&password=%s" .Release.Name .Values.postgresql.secret.user .Values.postgresql.secret.user }}
```
Comment (jpbu) [inline charts/managed-identity-wallets/templates/_helpers.tpl]: Shouldn't the last value be the password? It says `secret.user` twice

File: charts/managed-identity-wallets/values-pre-prod.yaml [hunk 2]
```diff
@@ -20,7 +20,7 @@ certificate:
   host: "managed-identity-wallets.pre-prod.demo.catena-x.net"
```
Comment (jpbu) [inline charts/managed-identity-wallets/values-pre-prod.yaml]: Do we still need the pre-prod values yaml file? I understood that those values are not used

File: charts/managed-identity-wallets/values.yaml [hunk 3]
```diff
@@ -218,7 +220,7 @@ datapool:
 managedIdentityWallets:
   secret:
     # -- Database connection string to the Postgres database of MIW
-    jdbcurl: "jdbc:postgresql://postgresql:5432/postgres?user=postgres&password=postgres"
+    jdbcurl: "jdbc:postgresql://managed-identity-wallets-postgresql:5432/postgres?user=postgres&password=postgres"
```
Comment (jpbu) [inline charts/managed-identity-wallets/values.yaml]: Do we still need this if it is created dynamically in the helpers.tpl? If only needed as a placeholder, we should comment it and change the value to a clear placeholder value

**Review by jpbu — COMMENTED** [review]
Overall looks good, thanks, a few comments and questions inline.

File: charts/managed-identity-wallets/templates/_helpers.tpl (hunk 1 above)
Comment (MajdT51) [reply charts/managed-identity-wallets/templates/_helpers.tpl]: yes it should be postgrespassword.

File: charts/managed-identity-wallets/values.yaml (hunk 3 above)
Comment (MajdT51) [reply charts/managed-identity-wallets/values.yaml]: a placeholder if a custom url is required

**Pull request merged**